style.css

```diff
@@ -0,0 +1,25 @@
+body {
+    background-color: orange;
+    margin: 10em;
+}
+#survey-form{
+    display: flex;
+    flex-direction: column;
+}
+#title{
+    text-align: center;
+}
+#description{
+    text-align: center;
+}
+button{
+    
+    margin-top: 16px;
+    background-color: royalblue;
+    border-radius: 10px;
+    width: 600px;
+    height: 100px;
+}
+#comment-section{
+    height: 400px;
+}
```

Comment:
Good job so far!  Lastly, the stylelint and webhint linters are not passing, so please fix the problems before asking for another review.